kurtosis/src/nodes/consensus/beacond/scripts/multiple-premined-deposits.sh

```
@@ -21,15 +21,15 @@
 
 
 /usr/bin/beacond init --chain-id $BEACOND_CHAIN_ID $BEACOND_MONIKER --home /tmp/config0/.beacond --consensus-key-algo $BEACOND_CONSENSUS_KEY_ALGO
-/usr/bin/beacond genesis add-premined-deposit --home /tmp/config0/.beacond
+/usr/bin/beacond genesis add-premined-deposit $DEPOSIT_AMOUNT $WITHDRAWAL_ADDRESS --home /tmp/config0/.beacond
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)